spec.md

```diff
@@ -814,6 +814,25 @@ So a pitch of `5` represent an `F4` and a pitch of `-2` represents an `A#3`.
 >
 > The pitches in this paragraph use [scientific pitch notation](https://en.wikipedia.org/wiki/Scientific_pitch_notation).
 
+Tools and games are expected to allow spaces to separate words both before and after the note text.
+
+E.g. before
+
+```abnf
+0 4 0 Hello|
+8 4 0  World|
+```
+
+and after
+
+```abnf
+0 4 0 Hello |
+8 4 0 World|
+```
+
+are treated the same. (the `|` is used to mark the end of the line for visualization purposes in the above example)
```

Comment:
```suggestion
are semantically equivalent. (the `|` is used to mark the end of the line for visualization purposes in the above example)
```

This goes a little beyond the rest of the paragraph. The semantics of spaces in notes are not really part of the spec yet, I think.
I don't think further explanation of the paragraph above is really necessary; the example itself would be enough for me.